### PR TITLE
Feature/maps id dependency

### PR DESCRIPTION
**Pull request by kennyw12**
Removes the id dependency on google maps

### CODE REVIEW

File: src/pivotal-ui/components/google-maps/google-maps.scss [hunk 1]
```diff
@@ -53,7 +53,8 @@ var maps = (function() {
   var mapOptions = {
     placeId: "ChIJ9w1pfYiAhYAR45k8AD-TjhA",
     center: new google.maps.LatLng(37.781787,-122.403911),
-    mapElementId: 'labs-map',
+    mapElementClass: 'labs-map',
+    mapElementId: 'google-maps',
```
Comment (stubbornella) [inline src/pivotal-ui/components/google-maps/google-maps.scss]: Did you need the id for something else?

**Comment by stubbornella** [comment]
I still see the id in there, but otherwise this looks good to me. @gpleiss - can you check it out?

**Comment by gpleiss** [comment]
Let's not merge this - we should fix the commit to mark the breaking change

**Comment by gpleiss** [comment]
Breaking change message added in the commit

**Pull request merged**